docs/content/en/docs/install/integrate.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Integrate KLT with your applications
+description: How to integrate the Keptn Lifecycle Toolkit into your Kubernetes cluster
+icon: concepts
+layout: quickstart
+weight: 45
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+Use Kubernetes annotations and labels
+to integrate the Keptn Lifecycle Toolkit into your Kubernetes cluster.
+
+The Keptn Lifecycle Toolkit monitors manifests
+that have been applied against the Kubernetes API
+and reacts if it finds a workload with special annotations/labels.
+This is a four-step process:
+
+* Annotate your workload(s)
+* Create a `KeptnApp` custom resource that references those workloads
```

Comment:
do we plan to add links to the appropriate pages here? we do not really explain these four steps in the next bit